### PR TITLE
reconfigure audit.

### PR DESCRIPTION
**Pull request by kekek2**

### CODE REVIEW

**Comment by fichtner** [comment]
I didn't touch this because it fell outside the audit scope, but many thanks for clearing up the code in this way too :)

File: www/web-proxy-useracl/src/etc/inc/plugins.inc.d/proxy_useracl.inc [hunk 1]
```diff
@@ -28,7 +28,7 @@
  *
  */
 
-function proxy_useracl_configure()
+function proxy_useracl_reload()
```
Comment (fichtner) [inline www/web-proxy-useracl/src/etc/inc/plugins.inc.d/proxy_useracl.inc]: I think this needs to stay as is

**Comment by kekek2** [comment]
https://github.com/opnsense/core/blob/master/src/opnsense/service/conf/actions.d/actions_proxy.conf#L31

**Comment by kekek2** [comment]
As I understand it, is now called 'reload' instead of 'reconfigure'
https://github.com/opnsense/core/commit/369a5e40fe278d2ab62f56f3a7c6cb2e5cd18382#diff-598394f22f42e7feae299851a2ae2c3c

**Comment by fichtner** [comment]
*_configure() is the plugin entry point from plugins.inc, can't rename this. reconfigure changes are necessary the backend only (configd)

File: www/web-proxy-useracl/src/etc/inc/plugins.inc.d/proxy_useracl.inc [hunk 2]
```diff
@@ -28,7 +28,7 @@
  *
  */
 
-function proxy_useracl_reload()
+function proxy_useracl_reconfigure()
```
Comment (fichtner) [inline www/web-proxy-useracl/src/etc/inc/plugins.inc.d/proxy_useracl.inc]: it's specifically "..._configure" as it was before :)

**Comment by fichtner** [comment]
merged, thanks!

**Pull request merged**